srcpkgs/apparmor/template

```diff
@@ -20,10 +20,24 @@ distfiles="https://launchpad.net/apparmor/${version%.*}/${version}/+download/app
 checksum=c6c161d6dbd99c2f10758ff347cbc6848223c7381f311de62522f22b0a16de64
 replaces="apparmor-vim>=0"
 
+case "$XBPS_TARGET_MACHINE" in
+	x86_64*|aarch64*|i686*)
+		configure_args+=" --with-python"
+		hostmakedepends+=" swig"
+		makedepends+=" python3-devel"
+		_have_python=yes
+	;;
+esac
+
 if [ -z "$CROSS_BUILD" ]; then
-	configure_args="--with-perl --with-python"
-	hostmakedepends+=" swig"
-	makedepends="python3-devel"
+	configure_args+=" --with-perl"
+	_have_perl=yes
+fi
+
+# /usr/lib/perl5/core_perl/CORE/perl.h:3360:22: error: unknown type name 'off64_t';
+if [ "$XBPS_TARGET_LIBC" = "musl" ]; then
+	CFLAGS+=" -D_LARGEFILE64_SOURCE"
+	CXXFLAGS+=" -D_LARGEFILE64_SOURCE"
 fi
```

Comment:
1. this should probably be `-D_FILE_OFFSET_BITS=64 -DLARGEFILE_SOURCE=1`
2. don't see why it can't be set on glibc too
3. don't need `+=`

Reply:
> 1. this should probably be `-D_FILE_OFFSET_BITS=64 -DLARGEFILE_SOURCE=1`

I don't think so. In <sys/types.h>, `off64_t` is behind `#if defined(_LARGEFILE64_SOURCE)`